mobile/components/Overlays/UserOverlay.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { ReactElement } from "react";
+import {
+  StyleSheet,
+  View,
+  ScrollView,
+  TouchableWithoutFeedback,
+  Keyboard,
+} from "react-native";
+import { Ionicons } from "@expo/vector-icons";
+import IconButton from "../IconButton";
+
+interface StepOverlayProps {
+  pageBody: ReactElement;
+  pageIcon?: ReactElement;
+}
+export default function LogOverlay({ pageBody }: StepOverlayProps) {
```

Comment:
rename StepOverlayProps to UserOverlayProps and remove the page icon field if you aren't using it